apps/website/src/routes/docs/headless/carousel/examples/vertical-direction.tsx

```diff
@@ -0,0 +1,27 @@
+import { component$, useStyles$ } from '@builder.io/qwik';
+import { Carousel } from '@qwik-ui/headless';
+
+export default component$(() => {
+  useStyles$(styles);
+
+  const colors = ['red', 'green', 'blue', 'yellow', 'purple', 'orange', 'pink'];
+
+  return (
+    <Carousel.Root class="carousel-root" gap={30} direction="column" maxSlideHeight={160}>
+      <div class="carousel-buttons">
+        <Carousel.Previous>Down</Carousel.Previous>
```

Comment:
This might be confusing. When the Up button is pressed, it goes to the next slide, but it's scrolling below to the next slide. (Downwards)

It looks to me that the names of these should be inverted. Swiping is done upwards to get to the next slide below, but not when programmatically changing the slide.

Reply:
ok, I will leave it as "Next" and "Prev" I think these names can work here as well.